blog/_posts/2019-06-18-diffeqbot.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  Hello @DiffEqBot
+author: <a href="http://github.com/kanav99">Kanav Gupta</a>
+---
+
+Hi! Today we all got a new member to the DiffEq family. Say Hi to our own [DiffEqBot](https://github.com/DiffEqBot) - A Bot which helps run benchmarks and compares with the current master of a given package. It also generates and stores the Reports generated in a [repository](https://github.com/DiffEqBot/Reports). What's special about this is that it is completely stateless (no databases involved at all, just juggling between repositories!) and it has no exposed public URLs. Even though highly inspired by Nanosoldier, this has a completely unique workflow.
```

Comment:
>
```suggestion
Hi! Today we all got a new member to the DiffEq family. Say hi to our own [DiffEqBot](https://github.com/DiffEqBot) - A bot which helps run benchmarks and compares with the current master of a given package. It also generates and stores the Reports generated in a [repository](https://github.com/DiffEqBot/Reports). What's special about this is that it is completely stateless (no databases involved at all, just juggling between repositories!) and it has no exposed public URLs. Even though highly inspired by Nanosoldier, this has a completely unique workflow.
```

Reply:
Is there a change here?

Reply:
Yeah I don't see any either

Reply:
„Hi“ should be in quotation , just not German ones

Reply:
One „Reports“ should be lower case

Reply:
Bot should be lowercase